com.unity.visualeffectgraph/Editor/Utilities/EventTester/VFXEventTesterWindow.cs

```diff
@@ -262,31 +274,34 @@ static void WindowGUI()
 
         static void SendEvent(string name)
         {
-            if (m_Effect == null) return;
-            var attrib = m_Effect.CreateVFXEventAttribute();
-            if (attrib == null) return;
-
-            // set all attributes
-            foreach (var attribute in m_Attributes)
+            if ((m_Effects?.Length).GetValueOrDefault(0) == 0) return;
+            foreach (var visualEffect in m_Effects)
             {
-                if (attribute == null) continue;
-                switch (attribute.type)
+                var attrib = visualEffect.CreateVFXEventAttribute();
+                if (attrib == null) return;
+
+                // set all attributes
+                foreach (var attribute in m_Attributes)
                 {
-                    case EventAttributeType.Bool: attrib.SetBool(attribute.name, (bool)attribute.value); break;
-                    case EventAttributeType.Float: attrib.SetFloat(attribute.name, (float)attribute.value); break;
-                    case EventAttributeType.Vector2: attrib.SetVector2(attribute.name, (Vector2)attribute.value); break;
-                    case EventAttributeType.Vector3: attrib.SetVector3(attribute.name, (Vector3)attribute.value); break;
-                    case EventAttributeType.Color: attrib.SetVector4(attribute.name, (Color)attribute.value); break;
+                    if (attribute == null) continue;
+                    switch (attribute.type)
+                    {
+                        case EventAttributeType.Bool: attrib.SetBool(attribute.name, (bool)attribute.value); break;
+                        case EventAttributeType.Float: attrib.SetFloat(attribute.name, (float)attribute.value); break;
+                        case EventAttributeType.Vector2: attrib.SetVector2(attribute.name, (Vector2)attribute.value); break;
+                        case EventAttributeType.Vector3: attrib.SetVector3(attribute.name, (Vector3)attribute.value); break;
+                        case EventAttributeType.Color: attrib.SetVector4(attribute.name, (Color)attribute.value); break;
+                    }
                 }
-            }
 
-            // then send event with attributes
-            if (name == "OnPlay")
-                m_Effect.Play(attrib);
-            else if (name == "OnStop")
-                m_Effect.Stop(attrib);
-            else
-                m_Effect.SendEvent(name, attrib);
+                // then send event with attributes
+                if (name == "OnPlay")
```

Comment:
You didn't change this code but these is a global static value for this string : https://docs.unity3d.com/ScriptReference/VFX.VisualEffectAsset.PlayEventName.html
You can replace with :
```
name == VisualEffectAsset.PlayEventName
```